1-client/views/home/home.js

```diff
@@ -7,34 +7,59 @@ app.controller('SequenceController', function ($scope, $http) {
 
 	function Sequencer () {
 		this.sequence = [];
+		var that = this;
```

Comment:
Some `that = this` required to maintain scope - can probably refactor later, but works for now.